nativeApp/bin/touch_tsx/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Command line utility for a creating a blank template for components
+ *
+ * Usage:
+ *  npm run touch:tsx [PATH] [FILE (optional)]
```

Comment:
Will this really work? Doesn't it need to be `npm run touch:tsx -- PATH [FILE]`

Also `[]` is usually used to indicate optional arguments.

Reply:
Yeah it works

Reply:
Also, I usually use [] to indicate variables rather than optional values. Here the first arg is required n the second is optional. Let me know if you have better way to show this.